File: src/test/java/net/yakclient/apitests/SecondMixinTestCase.java
package net.yakclient.apitests;

import net.yakclient.mixin.api.Injection;
import net.yakclient.mixin.api.InjectionType;
import net.yakclient.mixin.api.Mixer;

@Mixer(MixinSourceClassTest.class)
public class SecondMixinTestCase {
    @Injection(type = InjectionType.BEFORE_INVOKE)
    public void printTheString() {
        System.out.println("I also wanna print this");
    }
}

File: base.test/src/test/kotlin/net/yakclient/test/mixins/SecondMixinTestCase.kt
package net.yakclient.test.mixins

import net.yakclient.mixin.api.Injection
import net.yakclient.mixin.api.InjectionType
import net.yakclient.mixin.api.Mixer

@Mixer("net.yakclient.test.example.MixinSourceClassTest")
abstract class SecondMixinTestCase {
    @Injection//(type = InjectionType.BEFORE_RETURN)
    fun printTheString() {
        println("I also wanna print this")
    }
}